pkg/status/dist/templates/header.tmpl

```diff
@@ -31,6 +31,9 @@ NOTE: Changes made to this template should be reflected on the following templat
   ======
     {{- if .ntpOffset }}
     NTP offset: {{ humanizeDuration .ntpOffset "s"}}
+    {{- if ntpWarning .ntpOffset}}
+    {{displayNtpWarning}}
```

Comment:
Same as above. Since we need to call `color.YellowString` I think creating helpers like `yellowText`, `redText`, etc... would make sense